pipenv/project.py

```diff
@@ -240,12 +240,42 @@ def requirements_exists(self) -> bool:
         return bool(self.requirements_location)
 
     def is_venv_in_project(self) -> bool:
-        if self.s.PIPENV_VENV_IN_PROJECT is False:
-            return False
-        return self.s.PIPENV_VENV_IN_PROJECT or (
-            self.project_directory
-            and os.path.isdir(os.path.join(self.project_directory, ".venv"))
-        )
+        """
```

Comment:
I see a mix of docstrings and multi-line comments.  Does pipenv have a preference on the style/content of these comments?